src/ir/auth_logic/auth_logic_ast_traversing_visitor_test.cc

```diff
@@ -0,0 +1,79 @@
+//-----------------------------------------------------------------------------
```

Comment:
I don't think the files are formatted. Can you use clang-format with google3 style to format them?

Reply:
should be OK now.